client/branded/src/global-styles/grid.scss

```
@@ -0,0 +1,187 @@
+}
+$grid-row-columns: 6;
+$grid-columns: 12;
+// scss-docs-start grid-breakpoints
+$grid-breakpoints: (
+    xs: 0,
+    sm: 576px,
+    md: 768px,
+    lg: 992px,
+    xl: 1200px,
+    xxl: 1400px,
+);
+
+// Custom functions extracted from bootstrap
+@mixin media-breakpoint-up($name, $breakpoints: $grid-breakpoints) {
+    $min: map-get($breakpoints, $name);
+    @if $min != 0 {
+        @media (min-width: $min) {
+            @content;
+        }
+    } @else {
+        @content;
+    }
+}
+
+@function breakpoint-infix($name, $breakpoints: $grid-breakpoints) {
+    $min: map-get($breakpoints, $name);
+    @return if($min == 0, '', '-#{$name}');
+}
+
+// Single container class with breakpoint max-widths
+.container,
+  // 100% wide container at all breakpoints
+  .container-fluid {
+    width: 100%;
+    padding-right: calc(var(--container-padding-x) / 2); // stylelint-disable-line function-disallowed-list
+    padding-left: calc(var(--container-padding-x) / 2); // stylelint-disable-line function-disallowed-list
+    margin-right: auto;
+    margin-left: auto;
+}
+
+// Responsive containers that are 100% wide until a breakpoint
+@each $breakpoint, $container-max-width in $container-max-widths {
+    .container-#{$breakpoint} {
+        @extend .container-fluid;
+    }
+
+    @include media-breakpoint-up($breakpoint, $grid-breakpoints) {
+        %responsive-container-#{$breakpoint} {
+            max-width: $container-max-width;
+        }
+
+        // Extend each breakpoint which is smaller or equal to the current breakpoint
+        $extend-breakpoint: true;
+
+        @each $name, $width in $grid-breakpoints {
+            @if $extend-breakpoint {
+                .container#{breakpoint-infix($name, $grid-breakpoints)} {
+                    @extend %responsive-container-#{$breakpoint};
+                }
+
+                // Once the current breakpoint is reached, stop extending
+                @if $breakpoint == $name {
+                    $extend-breakpoint: false;
+                }
+            }
+        }
+    }
+}
+
+// Row
+// Rows contain your columns.
+.row {
+    display: flex;
+    flex-wrap: wrap;
+    margin-right: calc(-0.5 * var(--bs-gutter-x)); // stylelint-disable-line function-disallowed-list
+    margin-left: calc(-0.5 * var(--bs-gutter-x)); // stylelint-disable-line function-disallowed-list
+}
+
+// Remove the negative margin from default .row, then the horizontal padding
+// from all immediate children columns (to prevent runaway style inheritance).
+.no-gutters {
+    margin-right: 0;
+    margin-left: 0;
+
+    > .col,
+    > [class*='col-'] {
+        padding-right: 0;
+        padding-left: 0;
+    }
+}
+
+// Columns
+// Common styles for small and large grid columns
+%grid-column {
+    position: relative;
+    width: 100%;
+    padding-right: calc(var(--container-padding-x) / 2);
+    padding-left: calc(var(--container-padding-x) / 2);
+}
+
+@each $breakpoint in map-keys($grid-breakpoints) {
```

Let's leave a comment for reference from where this logic is extracted:

```suggestion
// Extracted from `bootstrap/scss/mixins/_grid-framework.scss`
@each $breakpoint in map-keys($grid-breakpoints) {
```